### ISSUE TITLE
Lab 7.1 add puppetlabs/sqlserver to the Puppetfile

### ISSUE THREAD

**Issue by josephoaks** [puppetlabs/ilt-practitioner-issues#16]
## Describe the Change You Would Like
This should be in the lab 2.1 and instructions to uncomment here

**Comment by golflimaechoecho** [comment]
in class 2260 sqlserver does appear to be added. Lab 7.1 fails on windows due to same issue with missing files as Lab 6.2 (https://github.com/puppetlabs/ilt-practitioner-issues/issues/15)

**Comment by golflimaechoecho** [comment]
Entry for `mod 'puppetlabs-sqlserver', '2.6.2'`is missing for class 2259 so perhaps I imagined/forgot about adding it for 2260!?

**Comment by greglarkin** [comment]
@golflimaechoecho @josephoaks This appears to be fixed, but can you verify and close the issue if you agree?

**Comment by golflimaechoecho** [comment]
@greglarkin I worked out what I did wrong last time :\

The Puppetfile entries are added under [Lab 2.1: Editing code](https://puppet-kmo.gitbook.io/puppet-practitioner/-MFBOjyVETcxjlTAR_Hx/table-of-content/lesson-2-editing-code-in-the-classroom/lab-2.1-editing-code)

The `puppetlabs-sqlserver` entry is present in the Windows version of the lab, but not Linux.

In theory this makes sense as the module is required for Windows, however it means if students want to switch between Linux and Windows or do both (as I was attempting), if they copied the Linux version in Lab 2.1 it would be missing.

Would it make sense to add the entry to the Linux lab as well so the `Puppetfile`s match/can be used interchangeably?
